mathesar/tests/api/test_record_api.py

```diff
@@ -241,7 +241,7 @@ def test_record_list_sort(create_patents_table, client):
     assert len(response_data['results']) == 50
 
     assert mock_get.call_args is not None
-    assert mock_get.call_args[1]['order_by'] == order_by
+    assert mock_get.call_args[1]['order_by'][:len(order_by)] == order_by
```

Comment:
Seems like an implementation detail. We do similar mocking in a few places and I'm not certain that it's beneficial. Just a note.